packages/analytics/amplify_analytics_pinpoint/example/web/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <title>amplify_analytics_pinpoint_example_app</title>
+</head>
+
+<body>
+<script defer src="sql-wasm.js"></script>
```

Comment:
*nit:* I might just add a comment to point to the drift docs in case someone wasn't sure why this was added.